ttt/symbols.rb

```diff
@@ -0,0 +1,24 @@
+class Symbols
+  @ai
+  @human
+  def initialize(ai_symbol, human_symbol)
+    @ai = ai_symbol
+    @human = human_symbol
+  end
+
+  def switch(symbol)
+    symbol == @ai ? @human : @ai
+  end
+
+  def get_human
+    @human
+  end
```

Comment:
Instead of a `get` method, Rubyists would use an accessor method. For example,

```rb
class Symbols
  attr_accessor :human

  def initialize(human_symbol)
    @human = human_symbol
  end
end

s = Symbols.new('x')
s.human #=> x
```